string pos1
string pos2
---
bool res